packages/cli/package.json

```diff
@@ -42,49 +43,44 @@
     "@ceramicnetwork/stream-tile": "^2.0.0-alpha.0",
     "@ceramicnetwork/streamid": "^2.0.0-alpha.0",
     "@stablelib/random": "^1.0.1",
-    "aws-sdk": "^2.1045.0",
+    "@types/abstract-leveldown": "~5.0.2",
+    "aws-sdk": "^2.1049.0",
     "commander": "^8.3.0",
     "cors": "^2.8.5",
     "dag-jose": "^1.0.0",
     "did-resolver": "^3.1.5",
     "dids": "^3.0.0-alpha.1",
     "ethr-did-resolver": "5.0.2",
-    "express": "^4.17.1",
+    "express": "^4.17.2",
     "flat": "^5.0.2",
-    "ipfs-http-client": "~54.0.2",
+    "get-port": "^5.1.1",
+    "go-ipfs": "^0.11.0",
+    "ipfs-http-client": "^55.0.0",
+    "ipfsd-ctl": "^10.0.5",
     "key-did-provider-ed25519": "^1.1.0",
     "key-did-resolver": "^2.0.0-alpha.0",
     "levelup": "^5.1.1",
     "morgan": "^1.10.0",
-    "nft-did-resolver": "^1.0.0",
+    "nft-did-resolver": "^2.0.0-alpha.1",
     "pkh-did-resolver": "^1.0.0-alpha.0",
     "reflect-metadata": "^0.1.13",
     "s3leveldown": "^2.2.2",
-    "safe-did-resolver": "^0.0.5",
+    "safe-did-resolver": "^1.0.0-alpha.0",
     "stream-to-array": "^2.3.0",
     "typedjson": "^1.8.0",
     "uint8arrays": "^3.0.0"
   },
   "devDependencies": {
+    "@ceramicnetwork/canary-integration": "^2.0.0-alpha.0",
```

Comment:
this seems backwards.  I thought the idea with canary-integration was that it could have a dependency on all our other packages and be a unified place for integration tests.  Putting this dependency here would prevent us from ever being able to import the `cli` package into a canary integration test without introducing a circular dependency.  Why do we need to add this?

Reply:
removed. not needed in any way.